### ISSUE TITLE
kerasの関数を理解する

### ISSUE THREAD

**Issue by kenkyu-kd** [kenkyu-kd/kenkyu#3]
https://qiita.com/cvusk/items/aa6270301ff2d14fb989

**Comment by kenkyu-kd** [comment]
keras.layyersのDense, Dropout, Activation, Flatten, Convolution2D, MaxPooling2D
